tests/phpunit/tests/Discovery.php

```diff
@@ -85,13 +85,16 @@ public static function wpTearDownAfterClass() {
 
 	public function setUp() {
 		parent::setUp();
+
 		$this->set_permalink_structure( '/%postname%/' );
 		$this->go_to( get_permalink( self::$story_id ) );
 	}
 
 	public function tearDown() {
+		$this->set_permalink_structure( '' );
```

Comment:
Can you explain this change?

Reply:
This is just some regular cleanup for these tests. The `setUp` calls `$this->set_permalink_structure( '/%postname%/' );`, so it's important to reset that again; otherwise it will affect other tests in other files.